client/src/containers/SignIn/SignIn.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import SmallContainer from '../../components/SmallContainer/SmallContainer';
+import './SignIn.css';
+
+function SignIn() {
+  return (
+    <SmallContainer>
+      <div className="has-text-centered">
+
+        <figure class="image is-128x128">
+          <img src="https://dummyimage.com/100x100/000/fff&text=CoderHood" alt="CoderHood" className="brand-logo is-rounded"/>
+        </figure>
+        <br/>
+
+        <div className="field has-text-left">
+          <label for="email" className="label is-text-left">Email</label>
+          <div className="control">
+            <input id="email" className="input email" type="email" placeholder="Email" />
```

Comment:
Create generic component so that other screen can use it too. This will help inconsistent UI among the components.

Reply:
Created `InpuField` component and refactored the code in commit [c9a0c3e](https://github.com/pesto-students/batch-8-CODERHOOD/commit/c9a0c3e9a98041bd76431987b3050799c848896c)